resorts/views/resort_list.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# General imports
+from collections import defaultdict
+from django.core.exceptions import ObjectDoesNotExist
+
+# Import Resort and ConditionsReport models
+from ..models import Resort, Conditions
+
+# Imports for using Google Maps
+import googlemaps
+import json
+
+# Google Maps requires a client key to work
+GMAPS = googlemaps.Client(key='AIzaSyBRrCgnGFkdRY-Z1hX6xaxoUFBczNI2664')
+
+# Constants
+NUMBER_TO_DISPLAY = 5
+DEFAULT_ORDER_ON = 'snow_in_past_24h'
+
+def get_resort_list(selected_resorts, user_address = "", number_to_display = NUMBER_TO_DISPLAY, order_on = DEFAULT_ORDER_ON):
+    """ Given a list of selected resorts, returns an ordered list of resorts and their conditions for display.
+        If a user_address is supplied, also returns driving time and distance information.
```

Comment:
I believe the second line and beyond of docstrings should line up with beginning of """ aka 1 tap stop. (Args is lined up correctly, and the args below it are rightfully indented)